webnative-examples/ditto
# Avivash/notifications

# Description

Adding notifications to the presets header element. I don't have different styles for the success/error types right now, but we can add those pretty easily if that's something we want(i just wasn't sure of the best way to differentiate them in the UI).

When a patch is created/updated, I chose not to list the patch name in the notification because those may potentially be quite long.

I've disabled notification stacking, so new notifications will now slide in from the side and replace the previous one

## Type of change


## Test plan (required)

- Update a patch, expect to see a success notification in the preset header
- Create a patch, expect to see a success notification in the preset header

## Screenshots/Screencaps

https://www.loom.com/share/43067c8ed24e46dfa4dccd088f2e5aee
